lab_3_generate_by_ngrams/main.py

```diff
@@ -104,6 +141,10 @@ def _put(self, element: str) -> None:
         In case of corrupt input arguments or invalid argument length,
         an element is not added to storage
         """
+        if not isinstance(element, str) or len(element) > 1:
+            return None
+        if self._storage.get(element) is None:
```

Comment:
`get` is unnecessary here, `in` is more appropriate

Reply:
i rewrited it :)